### PR TITLE
User profile update API

### PR DESCRIPTION
**Pull request by mthombare**
## Purpose
- Write API to Update user profile details

## Approach
- Written API to perform user profile update request

### Further Info
Ticket number: #46 
 
@yasmin2496  of the person or team responsible for reviewing proposed changes.

### CODE REVIEW

File: core/serializers.py [hunk 1]
```diff
@@ -152,6 +151,42 @@ def create(self, validated_data):
...
+        name = Organization.objects.filter(name=organization_name).first()
+        if name is not None:
+            instance.organization = name
+            instance.organization_name = name
+
+        instance.first_name = validated_data.get('first_name', instance.first_name)
+        instance.last_name = validated_data.get('last_name', instance.last_name)
+        instance.title = validated_data.get('title', instance.title)
+        instance.contact_info = validated_data.get('contact_info', instance.contact_info)
+        password = validated_data.get('password', None)
```
Comment (yasmin2496) [inline core/serializers.py:182]: Add survey status update here as well

**Review by yasmin2496 — CHANGES_REQUESTED** [review]
In profile user should be able to update survey_status as well. Rest is fine

**Comment by yasmin2496** [comment]
Also check that if request has no organization it throws an error. Org name should be optional

File: core/serializers.py (hunk 1 above)
Comment (mthombare) [reply core/serializers.py:184]: updated the fields

**Comment by yasmin2496** [comment]
Looks good. @MounikaReddy15 please make a note that you do any changes here pass org name as well

**Pull request merged**